ksqldb-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -574,7 +577,14 @@ private void printOverriddenProperties(final QueryDescription queryDescription)
 
   private void printQueryError(final QueryDescription query) {
     writer().println();
+
+    final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss.SSS");
```

Comment:
It's always worth including timezone info in the formatter. Otherwise, users are never quite sure if the output datetime is UTC, local time or something else.

Reply:
Done